locale/en/foundation/case-studies.md

```diff
@@ -63,6 +63,12 @@ interactive group.
 Enterprise conversations are short videos, recorded live, with large scale
 companies sharing their experiences with Node.js.
 
+## Node.js and Intel (Episode 13)
+
+Tracy Hinds and Mikeal Rogers chat with Monica Ene-Pietrosanu of Intel about how they use Node.js, and the company's efforts in contributing to open source technologies and diversity.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/SBR6WcSIqBE?list=PLfMzBWSH11xYNeUJXap4NTMX3TFtH-_0z" frameborder="0" allowfullscreen></iframe>
```

Comment:
you are correct, sir!  I knew it looked longer than the others, and that's why :)

want me to edit and resubmit?